tailwind.config.js

```diff
@@ -9,10 +13,28 @@ module.exports = {
   ],
   purge: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        mono: ["Jetbrains Mono", ...defaultTheme.fontFamily.sans],
+      },
+      colors: {
+        keto: colors.teal,
+        kratos: colors.orange,
+        hydra: colors.rose,
+        oauthkeeper: colors.pink,
+        info: colors.sky,
+        succes: colors.green,
+        warning: colors.yellow,
+        error: colors.red,
+      },
```

Comment:
If it's not used I would recommend removing that, it's easier to use `text-yellow` without additional layer.